shuup/xtheme/admin_module/__init__.py

```diff
@@ -43,6 +43,22 @@ def get_urls(self):  # doccov: ignore
                 "shuup.xtheme.admin_module.views.ThemeConfigDetailView",
                 name="xtheme.config_detail",
             ),
+            admin_url(
+                r"^xtheme/font/new/$",
```

Comment:
you need to create a view to list all the fonts

Reply:
done